elements/rh-table/rh-table.ts

```diff
@@ -60,6 +60,11 @@ export class RhTable extends LitElement {
     return this.querySelectorAll('tbody > tr') as NodeListOf<HTMLTableRowElement> | undefined;
   }
 
+  get #colHeaders(): NodeListOf<HTMLTableCellElement> | undefined {
+    return this.querySelectorAll(
+      'thead > tr > th') as NodeListOf<HTMLTableCellElement> | undefined;
```

Comment:
```suggestion
    return this.querySelectorAll<HTMLTableCellElement>('thead > tr > th');
```
you can assert in the call, and this function will never return `undefined`, so change the signature as well